src/components/Forms/Judge.jsx

```diff
@@ -151,6 +151,8 @@ const Judge = () => {
               user={judge}
               setUser={setJudge}
               text="Upload Photo"
+              sizeLimit="1 MB"
+              typeLimit=".png,.jpg,.jpeg"
```

Comment:
call it just types and make it an array, its slightly more processing for us, but its easier to use